### PR TITLE
added dev dockerfile & updated readme

### PR DESCRIPTION
**Pull request by cfoskin**

### CODE REVIEW

File: Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    mkdir -p /var/feedhenry/upload && \
+    chmod -R 755 /var/feedhenry && \
+    ln -sf /usr/share/zoneinfo/UTC /etc/localtime && \
+    mkdir -p config && \
+    chown -R default:root ./
+
+# Installing fonts to be able to render PDFs for submissions
+RUN yum install -y dejavu-sans-fonts
+RUN yum install -y http://cbs.centos.org/kojifiles/packages/phantomjs/1.9.7/3.el7/x86_64/phantomjs-1.9.7-3.el7.x86_64.rpm
+
```
Comment (PhilipGough) [inline Dockerfile.dev]: Could reduce the layers here using `&& \` but no big deal

File: Dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+USER default
+
+RUN	scl enable rh-nodejs4 "npm install -g nodemon" && \
+  scl enable rh-nodejs4 "npm install --production" && \
+  mv conf-docker.json config/conf.json && \
+  chmod -R ug+rw ./
+
```
Comment (PhilipGough) [inline Dockerfile.dev]: Small indentation fix here

**Comment by PhilipGough** [comment]
@cfoskin some comments but looks good. Lets wait for the dev script from @StephenCoady  and add that and I'll verify it at that point

**Comment by cfoskin** [comment]
@PhilipGough FYI not waiting on the dev script for this as other solution used.

**Comment by PhilipGough** [comment]
@cfoskin verifying this now. This needs to squash the commits and rebase before merging

**Comment by PhilipGough** [comment]
Verified in dev vm

**Pull request merged**